gcp/table_gcp_compute_instance.go

```diff
@@ -312,7 +312,11 @@ func listComputeInstances(ctx context.Context, d *plugin.QueryData, h *plugin.Hy
 	limit := d.QueryContext.Limit
 	if d.QueryContext.Limit != nil {
 		if *limit < *pageSize {
-			pageSize = limit
```

Comment:
Remove the line no 309  **plugin.Logger(ctx).Trace("listComputeInstances", "IAM I HERE")**